config/mycnf/mysql84.cnf

```diff
@@ -22,13 +22,13 @@ default_authentication_plugin = mysql_native_password
 #
 # VTTablet will enable semi-sync at the proper time when replication is set up,
 # or when a primary is promoted or demoted based on the durability policy configured.
-plugin-load = rpl_semi_sync_source=semisync_source.so;rpl_semi_sync_replica=semisync_replica.so
+plugin-load = rpl_semi_sync_master=semisync_master.so;rpl_semi_sync_slave=semisync_slave.so
```

Comment:
Are we sure that 8.4 still supports the old plugins and variable names?

Reply:
From the docs https://dev.mysql.com/doc/extending-mysql/8.4/en/writing-semisynchronous-replication-plugins.html it does look like they're available

Reply:
Just to be sure, I downloaded and deployed 8.4.1 via dbdeployer and verified that the plugins can indeed be loaded - 
```
mysql [localhost:8401] {msandbox} ((none)) > INSTALL PLUGIN rpl_semi_sync_source SONAME 'semisync_source.so';
Query OK, 0 rows affected (0.50 sec)

mysql [localhost:8401] {msandbox} ((none)) > UNINSTALL PLUGIN rpl_semi_sync_source;
Query OK, 0 rows affected (0.01 sec)

mysql [localhost:8401] {msandbox} ((none)) > INSTALL PLUGIN rpl_semi_sync_master SONAME 'semisync_m.so';
ERROR 1126 (HY000): Can't open shared library '/Users/manangupta/opt/mysql/8.4.1/lib/plugin/semisync_m.so' (errno: 2 dlopen(/Users/manangupta/opt/mysql/8.4.1/lib/plugin/semisync_m.so, 0x0002): tried: '/Users/manangupta/opt/mysql/8.4.1/lib/plugin)
mysql [localhost:8401] {msandbox} ((none)) > INSTALL PLUGIN rpl_semi_sync_master SONAME 'semisync_master.so';
Query OK, 0 rows affected, 1 warning (0.46 sec)

mysql [localhost:8401] {msandbox} ((none)) >
```

Reply:
It shows a warning, but still allows it - 
```sql
mysql [localhost:8401] {msandbox} ((none)) > INSTALL PLUGIN rpl_semi_sync_slave SONAME 'semisync_slave.so';
Query OK, 0 rows affected, 1 warning (0.45 sec)

mysql [localhost:8401] {msandbox} ((none)) > show warnings;
+-------+------+-----------------------------------------------------------------------------------------------------------------------+
| Level | Code | Message                                                                                                               |
+-------+------+-----------------------------------------------------------------------------------------------------------------------+
| Note  | 1287 | 'rpl_semi_sync_slave' is deprecated and will be removed in a future release. Please use rpl_semi_sync_replica instead |
+-------+------+-----------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)

mysql [localhost:8401] {msandbox} ((none)) >
```

Reply:
Why would we change the 8.4 specific config file? I thought that this issue was about Vitess downgrades.

Reply:
That's another way to handle this. Things probably don't even work with 8.4, and no one is using it, so we don't need to do anything with it in v20. The file is only present because the work of achieving compatibility with 8.4 has started, but not completed.

Reply:
What do we prefer? I don't mind reverting the change. I just thought it would be more consistent to say that in v20, we support both the plugins but continue to use the old ones everywhere. And starting v21, we'll have the new plugins being used.

Reply:
Let's revert the changes to 8.4 mycnf.

Reply:
Okay 👍

Reply:
Done.